audio/debug.c

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2009-2012 Chris McClelland
```

Comment:
Should this be part of `common`?

Reply:
Probably. Needs to have support for Opsis and Atlys though